dev/InfoBar/Strings/en-us/Resources.resw

```diff
@@ -133,8 +133,40 @@
     <value>InfoBar</value>
     <comment>This is the custom landmark used to denote an InfoBar to narrator.</comment>
   </data>
+  <data name="InfoBarIconSeverityErrorName" xml:space="preserve">
+    <value>Icon severity error</value>
+    <comment>Automation name used to announce the severity error icon.</comment>
+  </data>
+  <data name="InfoBarIconSeverityInformationalName" xml:space="preserve">
+    <value>Icon severity informational</value>
```

Comment:
Can each of these icon names be updated to "{Severity} icon"?

- "Informational icon"
- "Success icon"
- "Warning icon"
- "Error icon"

Reply:
@chingucoding before merging in, can you update these names?

Reply:
Oh yes, of course. Not sure how I missed this. should be updated now.